README.md

```diff
@@ -70,7 +70,7 @@ You can add your own extension in this list with your own github repository.
 | Add new extension to participate | | | |
 ## List of Docker Extensions suggestions
 
-Here is a list of suggestions or ideas for new Docker Extensions:
+Here is a list of suggestions or ideas for new Docker Extensions you may need;
```

Comment:
“you may need” not required. The readme suggests the user to pick from the list of ideas.